container-search/src/main/java/com/yahoo/search/dispatch/rpc/RpcResourcePool.java

```diff
@@ -19,43 +27,70 @@
     /** The compression method which will be used with rpc dispatch. "lz4" (default) and "none" is supported. */
     public final static CompoundName dispatchCompression = new CompoundName("dispatch.compression");
 
-    private final Compressor compressor = new Compressor();
-    private final Client client;
+    private final Compressor compressor = new Compressor(CompressionType.LZ4, 5, 0.95, 32);
```

Comment:
Have you benchmarked different LZ4 compression levels ?
I think that when I benchmarked last time 4 years+ 3 was the golden number.
searchcore/src/vespa/searchcore/config/fdispatchrc.def:packetcompresslevel int default = 3 restart
searchcore/src/vespa/searchcore/config/proton.def:packetcompresslevel int default=3

Reply:
com.yahoo.compress.Compressor uses 7 as a threshold to pick either fastCompressor() or highCompressor() from LZ4Factory, so I tested only those. But I'll give highCompressor(3) a shot since the JRT code path is still 10-15% slower than FS4 (even without TLS)

Reply:
highCompressor(3) is observably slower than fastCompressor() as expected